tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1335,17 +1334,17 @@ static void VerifyPdbUtil(
   {
     CComBSTR str;
     VERIFY_SUCCEEDED(pPdbUtils->GetTargetProfile(&str));
-    VERIFY_ARE_EQUAL(str, L"ps_6_0");
+    VERIFY_ARE_EQUAL_WSTR(str.m_str, L"ps_6_0");
```

Comment:
Swapping order will result in a message that prints the `str.m_str` when this fails instead of re-printing the reference string.

Reply:
Missed one :(

```suggestion
    VERIFY_ARE_EQUAL_WSTR(L"ps_6_0", str.m_str);
```